elasticache-redis.tf

```diff
@@ -0,0 +1,130 @@
+variable "redis_instance_type" {
+  type        = "string"
+  default     = "cache.t2.medium"
+  description = "EC2 instance type for Redis cluster"
+}
+
+variable "redis_cluster_size" {
+  type        = "string"
+  default     = "2"
+  description = "Redis cluster size"
+}
+
+variable "redis_cluster_enabled" {
+  type        = "string"
+  default     = ""
```

Comment:
```suggestion
  default     = "true"
```

Reply:
@goruha This is not needed and will fall back on `enabled` for the whole module if we don't override a specific service enabled flag like this. It was done like this so we could turn enable/disable individual services if we needed, rather than the entire module enable/disabled